sdk/synapse/Azure.ResourceManager.Synapse/samples/Generated/Samples/Sample_SynapseDatabaseResource.cs

```diff
@@ -171,7 +171,7 @@ public async Task ValidateDataConnection_KustoPoolDataConnectionValidation()
                 DataConnectionName = "DataConnections8",
                 Properties = new SynapseEventHubDataConnection(),
             };
-            ArmOperation<SynapseDataConnectionValidationListResult> lro = await synapseDatabase.ValidateDataConnectionAsync(WaitUntil.Completed, synapseDataConnectionValidation);
+            ArmOperation<SynapseDataConnectionValidationListResult> lro = await synapseDatabase.DataConnectionValidationKustoPoolDataConnectionAsync(WaitUntil.Completed, synapseDataConnectionValidation);
```

Comment:
mapping failed: https://github.com/Azure/azure-sdk-for-net/blob/91af14e56d8b9f12d66fc5217e2e13670a21d8e1/sdk/synapse/Azure.ResourceManager.Synapse/src/autorest.md?plain=1#L382